packages/core/src/governance/server/actions.ts

```diff
@@ -86,8 +86,18 @@ export async function updateTokenAction(
   input: UpdateTokenInput,
   { authToken }: { authToken: string },
 ) {
-  if (!authToken) throw new Error('authToken is required to update token');
-  return updateToken(input, { db });
+  if (!authToken) {
+    console.error('authToken is required to update token');
+    throw new Error('authToken is required to update token');
+  }
+
+  try {
+    const result = await updateToken(input, { db });
+    return result;
+  } catch (error) {
+    console.error('updateTokenAction failed:', error);
+    throw error;
+  }
```

Comment:
_optional_
Ideally only throw an error and leave logging to a caller
```suggestion
  if (!authToken) {
    throw new Error('authToken is required to update token');
  }

  try {
    const result = await updateToken(input, { db });
    return result;
  } catch (error) {
    throw Error('updateTokenAction failed:', error);
  }
```